greynoise/icon_greynoise/util/util.py

```diff
@@ -0,0 +1,25 @@
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class GNRequestFailure(Exception):
+    def __init__(self, status_code, message):
+        self.status_code = status_code
+        self.message = message['message']
+
+        raise PluginException(
+            cause="Received HTTP %d status code from GreyNoise. Verify your input and try again." % self.status_code,
+            assistance="If the issue persists please contact support.",
```

Comment:
```suggestion
            assistance="If the issue persists please contact support.",
```
This should likely refer the user to GreyNoise customer support since this is not a Rapid7-developed plugin. @jschipp-r7 ?

Reply:
Thanks!